great_expectations/expectations/expectation.py

```diff
@@ -2188,8 +2172,9 @@ def _get_augmented_library_metadata(self):
         augmented_library_metadata["problems"] = problems
         return AugmentedLibraryMetadata.from_legacy_dict(augmented_library_metadata)
 
+    @staticmethod
     def _get_maturity_checklist(
-        self,
+        expectation_instance,
```

Comment:
See note in `build_gallery.py` about potentially reverting this to `self` and calling these methods on the instance itself.